pyccel/codegen/wrapper/c_to_python_wrapper.py

```diff
@@ -1498,7 +1498,7 @@ def _wrap_FunctionDef(self, expr):
             func_args = [FunctionDefArgument(a) for a in func_args]
             body = []
         else:
-            if in_interface or original_func_name in magic_binary_funcs or original_func_name == '__len__':
+            if in_interface or original_func_name in magic_binary_funcs or original_func_name in ('__len__', '__getitem__'):
```

Comment:
Isn't `mp_subscript` a binary function?

Reply:
Sorry, I just got what you meant.
[fixed](https://github.com/pyccel/pyccel/pull/2219/commits/5e755a8574dbd850e45d3055bb66a5f3590261c8)